scss/tokens/_component.scss

```diff
@@ -310,14 +284,32 @@ $ouds-link-size-min-height-default: $ouds-size-min-interactive-area !default;
 $ouds-link-size-min-height-small: $core-ouds-dimension-550 !default;
 $ouds-link-size-min-width-default: $ouds-size-min-interactive-area !default;
 $ouds-link-size-min-width-small: $core-ouds-dimension-550 !default;
-$ouds-link-space-column-gap-arrow-default: $ouds-space-column-gap-2xs !default;
-$ouds-link-space-column-gap-arrow-small: $ouds-space-column-gap-2xs !default;
+$ouds-link-space-column-gap-chevron-default: $ouds-space-column-gap-2xs !default;
+$ouds-link-space-column-gap-chevron-small: $ouds-space-column-gap-2xs !default;
```

Comment:
Why do we change chevron in here and keep the other tokens named arrow ? e.g. `$ouds-link-color-arrow-enabled`, ...
This might impact our link CSS variables API.